maps/randomvaults/objects.dm

```diff
@@ -46,6 +46,8 @@
 
 /area/vault/asteroid
 
+/area/vault/asteroidfield
```

Comment:
As there's nothing really going on in here, why not use the auto-gen area?

Reply:
Shrug.  Habit?  Fear of some code kraken coming from the inky blackness and eating my map.